lib/Kconfig.kfence

```diff
@@ -35,6 +35,14 @@ config KFENCE_NUM_OBJECTS
 	  pages are required; with one containing the object and two adjacent
 	  ones used as redzones.
 
+config KFENCE_FAULT_INJECTION
+	bool "Fault injection for stress testing"
```

Comment:
Shall we make this parameter an int that we can pass to `prandom_u32_max()` instead?

Reply:
Yes, good idea. I'll add a commit.